cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1321,6 +1331,22 @@ func getEnableBackendKeepalive(service *v1.Service) (bool, error) {
 	return enableBackendKeepalive, nil
 }
 
+func getHttpIdleTimeoutSeconds(service *v1.Service) (*uint64, error) {
+	httpIdleTimeoutSeconds, ok := service.Annotations[annDOHttpIdleTimeoutSeconds]
+
+	if !ok {
+		return nil, nil
+	}
+
+	httpIdleTimeout, err := strconv.ParseUint(httpIdleTimeoutSeconds, 10, 64)
+
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse LB http_idle_timeout_seconds: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to parse LB http_idle_timeout_seconds: %w", err)
```

Reply:
Do we actually intend to parse this error? If not or if it's unclear at this moment, then I think we should keep it at `%s` to not extend the API surface area unnecessarily per Go's recommendation: https://go.dev/blog/go1.13-errors#whether-to-wrap

Reply:
fair point 👍